Readability.js

```diff
@@ -1655,8 +1655,6 @@ Readability.prototype = {
           toRemove = true;
         } else if (!isList && weight < 25 && linkDensity > 0.2) {
           toRemove = true;
-        } else if (weight >= 25 && linkDensity > 0.5) {
-          toRemove = true;
```

Comment:
I don't have a counterexample to hand, no. I am just really uncomfortable making sweeping changes like this *especially* when we don't have a good understanding of why the code is there, and considering that if we then later find out this breaks something, we won't necessarily be in a position to readjust.

I think the top candidate extraction isn't really that helpful here - if we select a container that is too high up the DOM tree, or if there are "related" links or something that come in, this will previously have stripped them and will now have kept them.